app/jobs/broadcast_compatible_targets_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
Why this? You are not handling strings here

Reply:
Rubocop automatically adds it.